JsonSchema/JsonSchema.cs

```diff
@@ -91,12 +92,96 @@ public class JsonSchema : IBaseDocument
 	internal Dictionary<string, (JsonSchema Schema, bool IsDynamic)> Anchors { get; } = [];
 	internal JsonSchema? RecursiveAnchor { get; set; }
 
+	internal static JsonSerializerOptions SerializerOptions
+	{
+		get
+		{
+			lock (_serializerOptionsLock)
+			{
+				EnsureTypeInfoResolver();
+				_serializerOptions ??= new JsonSerializerOptions
+				{
+#if NET8_0_OR_GREATER
+					TypeInfoResolver = _typeInfoResolver
+#endif
+				};
+
+				return _serializerOptions!;
+			}
+		}
+	}
+
+	internal static JsonSerializerOptions SerializerOptionsUnsafeRelaxedJsonEscaping
```

Comment:
It doesn't look like these things actually have to do with the `JsonSchema` class specifically.  I'd like them removed into a separate (static) class, please.

Reply:
Sure.

Reply:
Would you be ok if they're on the JsonSchemaSerializerContext?

Reply:
Yeah, if that won't interfere with anything else.